docs/configuration-rum.asciidoc

```diff
@@ -100,6 +100,17 @@ custom values configured here are appended to the default list and used as the v
 
 Default: `[]`
 
+[float]
+[[rum-response-headers]]
+==== `response_headers`
+Custom HTTP headers to add to RUM responses.
+This can be useful for security policy compliance.
+
+Any values set in `apm-server.response_headers` also apply to RUM events.
```

Comment:
I don't quite understand what you mean with this sentence; custom headers are not added to the indexed trace events. Can we just remove this line?

Reply:
Sure, I can remove it. This sentence was added based on what I read in https://github.com/elastic/apm-server/pull/4523#issue-539075612

> If both are defined, then they will both be added to RUM responses, with values for the same key concatenated.

Reply:
```suggestion
```